core/modules/user/user.module

```diff
@@ -1630,6 +1631,19 @@ function user_menu_alter(&$items) {
   }
 }
 
+/**
+ * Return the user page tabs setting.
+ */
+function user_login_page_tabs() {
+  static $user_login_page_tabs;
```

Comment:
Why a static cache? Is this supposed to be called many times per page request?

Reply:
Agreed not needed, see below comment.